examples/pendulum.jl

```diff
@@ -0,0 +1,49 @@
+using Gadfly
+
+g = 9.807 # m/s^2
+
+τ = 2*pi
+δt = 0.03
+
+type Pendulum
+    θ :: Real
+    ω :: Real
+end
+
+# Euler integration (unused)
+function Δ(s::Pendulum,δt,τ) # s is for system
+    θ=s.θ+s.ω*δt
+    ω=s.ω+τ(s)*δt
+    return Pendulum(θ,ω)
+end
+
+function midpoint_step ( s , dt :: Real, τ)
+    ω_half = s.ω + 0.5*dt*τ(s)
+    θ_half = s.θ + 0.5*dt*s.ω 
+    θ      = s.θ + dt*ω_half
+    ω      = s.ω + dt*τ(Pendulum(θ_half,ω_half))
+    return θ, ω
+end
+
+acc(s) = -g/1*sin(s.θ)
+
+function step(s,δt,acc)
+    th, w = midpoint_step(s,δt,acc)
+    return Pendulum(th, w)
+end
+
+s0 = Pendulum(τ/4-0.4, 4.000)
+
+function main(window)
+    push!(window.assets, "widgets")
+
+    fps10 = fps(25) # lies
```

Comment:
Can you use fpswhen(window.alive, 25) here? This computation will continue to run after the window closes otherwise.